sql/org_mozilla_firefox_derived/migrated_clients_v1/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+friendly_name: Migrated Clients
+description: >
+  Contains the Fennec Client ID and Fenix Client ID for any clients
+  who have migrated from Fennec to Fenix. This is across all release
+  channels. Every client will have at most one entry, and if a 
+  Fennec client is not present then it indicates that they have
+  not migrated. Fennec Client ID, channel, and submission_date
+  are taken from the _last_ value seen for that client.
+owners:
+  - frank@mozilla.com
+labels:
+  application: fenix
+  schedule: daily
+  incremental: true
```

Comment:
I'm wondering if this is indeed incremental.

Reply:
If we're rewriting it every day, then I'd say it's not incremental.

Reply:
Hm, but we use the results from the previous day, and increment the entire table based on those.